packages/ocaml-variants/ocaml-variants.5.0.0~beta1+options/opam

```diff
@@ -0,0 +1,79 @@
+opam-version: "2.0"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+synopsis: "First beta release of OCaml 5.0.0"
+maintainer: "platform@lists.ocaml.org"
+authors: ["Xavier Leroy" "Damien Doligez" "Alain Frisch" "Jacques Garrigue" "Didier Rémy" "Jérôme Vouillon"]
+homepage: "https://ocaml.org"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git+https://github.com/ocaml/ocaml.git#5.0"
+depends: [
+  "ocaml" {= "5.0.0" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-domains" {post}
+  "base-nnp" {post}
+  "ocaml-option-bytecode-only" {arch != "arm64" & arch != "x86_64"}
```

Comment:
if we're adding that, shouldn't there be a similar case for `ocaml-base-compiler` ?

Reply:
That sounds reasonable: to me:  the change originated from the 5.0.0+trunk opam package that was updated by @dra27 but I never imported the change to base-compiler package.

Reply:
This is a good idea, but it will need the vanilla package to be tweaked. The `+options` package definitely needs it - I only didn't update the alpha packages in #21510 because of the unnecessary rebuild that would have triggered for existing switches.

Reply:
Do we want to tweak the vanilla package now or should we wait for the next beta?